torch_xla/experimental/custom_kernel.py

```diff
@@ -222,7 +265,17 @@ def shape_dtype(q, *arg):
           min(FlashAttention.DEFAULT_BLOCK_SIZES["block_k"], k.shape[2]),
           False,
           static_argnums=range(5, 13))
+
+      if segment_ids is None:
+        o = torch_xla._XLAC._xla_tpu_custom_call([q, k, v], payload, shapes,
+                                                 dtypes)
+
+      else:
+        o = torch_xla._XLAC._xla_tpu_custom_call(
+            [q, k, v, q_segment_ids, kv_segment_ids], payload, shapes, dtypes)
+
       if not save_residuals:
+        o = o[0]
```

Comment:
what's this for?

Reply:
_xla_tpu_custom_call always return an array.